hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/AsyncResponsePacketHandler.java

```diff
@@ -97,40 +98,26 @@ public ResponseThread(HazelcastThreadGroup threadGroup,
         public void run() {
             try {
                 doRun();
+            } catch (InterruptedException e) {
+                ignore(e);
```

Comment:
Maybe an explicit `return` is a better idea? The exception isn't literally "ignored", it's being _handled_ by ending the runnable.